tests/onnxruntime/test_optimization.py

```diff
@@ -167,9 +208,13 @@ def preprocess_function(examples, tokenizer):
 
         with tempfile.TemporaryDirectory() as tmp_dir:
             output_dir = Path(tmp_dir)
-            model_path = output_dir.joinpath("model.onnx")
+            model = ORTModelForSequenceClassification.from_pretrained(model_name, from_transformers=True)
+            model.save_pretrained(tmp_dir)
+            tokenizer = AutoTokenizer.from_pretrained(model_name)
             q8_model_path = output_dir.joinpath("model-quantized.onnx")
-            quantizer = ORTQuantizer.from_pretrained(model_name, feature="sequence-classification")
+
+            quantizer = ORTQuantizer.from_pretrained(model)
+
             calibration_dataset = quantizer.get_calibration_dataset(
                 "glue",
                 dataset_config_name="sst2",
```

Comment:
I cannot comment it at the right place because it has not been modified, but the line just below this one is:
```python
preprocess_function=partial(preprocess_function, tokenizer=quantizer.preprocessor),
```
and I think it should be
```python
preprocess_function=partial(preprocess_function, tokenizer=tokenizer),
```